resources/web/style/admonishment.pcss

```diff
@@ -0,0 +1,38 @@
+#guide .Admonishment {
+  position: relative;
+  font-size: 16px;
+
+  &:hover {
+    color: black;
+    transition: all 0.3s ease;
+    .Admonishment-detail {
+      z-index: 10000;
+      left: -5em;
+      opacity: 1;
+      color: #555;
+      transition: color 0.5s ease;
+    }
+  }
+
+  &-title, &-version {
+    font-family:Monaco, 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', 'Lucida Console', monospace!important;
```

Comment:
Just something to think about, generalized classes like maybe `u-monotype` or something that does the font-family and font-size may be helpful. We could end up with a bucket of general classes that can be combined for the effects we want in templates.

Nothing actionable, just something to think about when looking at CSS definitions.

Reply:
++